src/rebar_compiler_erl.erl

```diff
@@ -3,7 +3,7 @@
 -behaviour(rebar_compiler).
 
 -export([context/1,
-         needed_files/3,
+         needed_files/4,
```

Comment:
is this a change in custom compilers such as mix and whatnot as well?

Reply:
Actually mix doesn't need this. mix is a project builder, not a compiler.